selfdrive/ui/qt/maps/map_settings.h

```diff
@@ -23,6 +23,21 @@ const QString NAV_FAVORITE_LABEL_WORK = "work";
 class NavDestination;
 class DestinationWidget;
 
+class NavigationRequest : public QObject {
+  Q_OBJECT
+
+public:
+  NavigationRequest(QObject *parent);
```

Comment:
should we make the constructor non-public to avoid doing this again accidentally?